### PR TITLE
feat(cxl-ui): align course cards horizontally

### PR DESCRIPTION
**Pull request by paulkirspuu**

### CODE REVIEW

File: packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss [hunk 1]
```diff
@@ -41,11 +41,15 @@ cxl-vaadin-accordion {
       word-break: break-word;
     }
 
-    cxl-accordion-card[opened] {
-      .entry-title {
-        overflow: visible;
+    cxl-accordion-card { display: table; margin: 0;
```
Comment (lkraav) [inline packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss:44]: We are most certainly not doing `display: table`

**Comment by paulkirspuu** [comment]
origin/raphael-accordion-issue48-grid:

image

origin/paul:

image

File: packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss (hunk 1 above)
Comment (paulkirspuu) [reply packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss:44]: ok ill find an alternative.

**Comment by lkraav** [comment]
* This is a `refactor()` commit, not `feat()`
* Commit doesn't list which issue it wants to fix (in this case #48)

File: packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@ cxl-vaadin-accordion {
   /**
    * Theme "cxl-accordion-card".
    */
-  &[theme~="cxl-accordion-card"] {
+  &[theme~="cxl-accordion-card"] { display: grid; grid-gap: var(--lumo-space-m); grid-template-columns: repeat( auto-fill, minmax( calc( var(--cxl-wrap-width) / 4 ), 1fr ) );
```
Comment (lkraav) [inline packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss:8]: Prettier did not auto-break it into Aybolit's multiple lines formatting :thinking: Can you update manually?

**Comment by pawelkmpt** [comment]
@paulkirspuu @lkraav it's old. can we close it?

**Comment by lkraav** [comment]
Yeah it should be a feature branch, but I think current live solution is still only living in theme CSS, so it should be upgraded to CXL UI level. However we do it, can probably continue in #53 then.

**Pull request closed without merging**